modules/authentication/proc-authenticationg-with-the-guest-user-on-a-helm-based-installation.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: PROCEDURE
+[id="authenticating-with-the-guest-user-on-a-helm-based-installation_{context}"]
+= Authenticating with the Guest user on a Helm-based installation
+
+On a Helm-based installation, you can configure {product-short} to log in as a Guest user and access {product-short} features.
```

Comment:
More for my edification:

When writing Helm instructions, we say "On a Helm-based installation."

When writing Operator instructions, we say "After an Operator-based installation," correct?

Reply:
Is it confusing?

That was intended to express a general nuance, without putting to much emphasis on it (because it should be acquired knowledge for the user at this point):

With Helm, you can configure before the installation, or reconfigure afterwards.

With the Operator, you need to install first, then you can configure the running instance.